components/brave_wallet_ui/components/desktop/popup-modals/add-account-modal/add-imported-account-modal.tsx

```diff
@@ -161,6 +170,22 @@ export const ImportAccountModal = () => {
     copyToClipboard('')
   }, [])
 
+  const onClickFileUpload = React.useCallback(() => {
+    // To prevent panel from being closed when file chooser is open
+    if (isPanel) {
+      dispatch(PanelActions.setCloseOnDeactivate(false))
+      // For resume close on deactive when file chooser is close(select/cancel)
+      window.addEventListener('focus', onFocusFileUpload)
+    }
+  }, [])
```

Comment:
Doesn't need to memoized in a `useCallback`

```suggestion
  const onClickFileUpload = () => {
    // To prevent panel from being closed when file chooser is open
    if (isPanel) {
      dispatch(PanelActions.setCloseOnDeactivate(false))
      // For resume close on deactive when file chooser is close(select/cancel)
      window.addEventListener('focus', onFocusFileUpload)
    }
  }
```

Reply:
fixed and squashed.